packages/validation/frontend/sirius-components-validation/package.json

```diff
@@ -34,12 +34,10 @@
     "@eclipse-sirius/sirius-components-core": "*",
     "@mui/material": "5.15.19",
     "@mui/icons-material": "5.15.19",
-    "@xstate/react": "3.0.0",
     "graphql": "16.8.1",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "tss-react": "4.9.7",
-    "xstate": "4.32.1"
+    "tss-react": "4.9.7"
```

Comment:
Why? `sirius-components-core` depends on `xstate` so it needs to be referenced as a peer/dev dependency in `sirius-components-validation`.